tests/test_spring_boot_admin_registration.py

```diff
@@ -84,3 +85,11 @@ def get_registration_handler(registration_url: str, registration_auth: Optional[
         service_url="http://whatever/service",
         registration_interval_sec=100
     )
+
+
+def _start_registration(registration_handler: BootAdminRegistrationHandler) -> None:
+    # Registration is done asynchronously, for the test, ask to register shortly after start is called
+    registration_handler.start(0.01)
+
+    # Wait enough after starting the registration allowing the async registration to happen.
+    time.sleep(0.1)
```

Comment:
Is 100ms enough to complete the registration?

Reply:
in the test, yes, because the registration-interval is set to 10ms.
ideally we could make `registration_handler.start(0.01)` be async but I think that's an overkill.